### ISSUE TITLE
Make html fails to complete

### ISSUE THREAD

**Issue by Omasiri** [odoo/documentation-user#213]
After downloading the repo, I navigated to the root dir of 'documentation-user' and executed 'make html'.

Here's the error message:
```
/home/documentation-user/accounting/payables/misc/employee_expense.rst:210: ERROR: Error in "todo" directive:
unknown option: "doc".

.. todo:: :doc: ../../../sale/invoicing/service/expense
/home/documentation-user/inventory/routes/costing/landed_costs.rst:3: WARNING: Could not find any field related to the action [stock_landed_costs.stock_landed_cost_type_action]
/home/documentation-user/legal/terms/partnership.rst:220: WARNING: duplicate label appendix_a, other instance in /home/documentation-user/legal/terms/enterprise.rst
looking for now-outdated files... none found
pickling environment... done
checking consistency... /home/documentation-user/expense.rst:: WARNING: document isn't included in any toctree
/home/documentation-user/legal/terms/enterprise_tex.rst:: WARNING: document isn't included in any toctree
/home/documentation-user/legal/terms/i18n/enterprise_tex_fr.rst:: WARNING: document isn't included in any toctree
/home/documentation-user/legal/terms/partnership_tex.rst:: WARNING: document isn't included in any toctree
done
preparing documents... done
writing output... [  4%] accounting/bank/setup/create_bank_account              
Exception occurred:
  File "/usr/local/lib/python2.7/dist-packages/sphinx/writers/html.py", line 223, in visit_reference
    assert len(node) == 1 and isinstance(node[0], nodes.image)
AssertionError
The full traceback has been saved in /tmp/sphinx-err-0k491Y.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
Makefile:60: recipe for target 'html' failed
make: *** [html] Error 1
```

**Comment by tivisse** [comment]
Hello @Omasiri 

Unfortunately, I'm not able to reproduce your issue. Can you please provide the full traceback that has been saved in /tmp/sphinx-err-0k491Y.log.

Thanks for the feedback.

**Comment by tivisse** [comment]
I've tested it with sphinx-build on version 1.4 and 1.3.1 and it's working fine. Can you also provide your sphinx version ?

Thanks.

**Comment by tivisse** [comment]
@Omasiri I reopen the ticket as the issue actually occurs with Sphinx 1.5.5

**Comment by mart-e** [comment]
@xmo-odoo could you check? Looks like it is a problem in the translation of the `demo:fields` tag

```
# Sphinx version: 1.5.5
# Python version: 2.7.13 (CPython)
# Docutils version: 0.13.1 release
# Jinja2 version: 2.9.6
# Last messages:
#   writing output... [  2%] accounting/bank/feeds/qif
#   writing output... [  2%] accounting/bank/feeds/synchronize
#   writing output... [  2%] accounting/bank/misc
#   writing output... [  2%] accounting/bank/misc/batch
#   writing output... [  3%] accounting/bank/misc/interbank
#   writing output... [  3%] accounting/bank/reconciliation
#   writing output... [  3%] accounting/bank/reconciliation/configure
#   writing output... [  3%] accounting/bank/reconciliation/use_cases
#   writing output... [  4%] accounting/bank/setup
#   writing output... [  4%] accounting/bank/setup/create_bank_account
# Loaded extensions:
#   embedded_video (unknown version) from /home/mat/odoo/documentation-user/_extensions/embedded_video.pyc
#   sphinx.ext.todo (1.5.5) from /usr/lib/python2.7/site-packages/sphinx/ext/todo.pyc
#   github_link (unknown version) from /home/mat/odoo/documentation-user/_extensions/github_link.pyc
#   alabaster (0.7.10) from /usr/lib/python2.7/site-packages/alabaster/__init__.pyc
#   html_domain (unknown version) from /home/mat/odoo/documentation-user/_extensions/html_domain.pyc
#   sphinx.ext.ifconfig (1.5.5) from /usr/lib/python2.7/site-packages/sphinx/ext/ifconfig.pyc
#   demo_link (unknown version) from /home/mat/odoo/documentation-user/_extensions/demo_link.pyc
#   odoo (unknown version) from /home/mat/odoo/documentation-user/_extensions/odoo/__init__.pyc
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/sphinx/cmdline.py", line 296, in main
    app.build(opts.force_all, filenames)
  File "/usr/lib/python2.7/site-packages/sphinx/application.py", line 333, in build
    self.builder.build_update()
  File "/usr/lib/python2.7/site-packages/sphinx/builders/__init__.py", line 251, in build_update
    'out of date' % len(to_build))
  File "/usr/lib/python2.7/site-packages/sphinx/builders/__init__.py", line 322, in build
    self.write(docnames, list(updated_docnames), method)
  File "/usr/lib/python2.7/site-packages/sphinx/builders/__init__.py", line 360, in write
    self._write_serial(sorted(docnames), warnings)
  File "/usr/lib/python2.7/site-packages/sphinx/builders/__init__.py", line 368, in _write_serial
    self.write_doc(docname, doctree)
  File "/usr/lib/python2.7/site-packages/sphinx/builders/html.py", line 455, in write_doc
    self.docwriter.write(doctree, destination)
  File "/usr/lib/python2.7/site-packages/docutils/writers/__init__.py", line 80, in write
    self.translate()
  File "/usr/lib/python2.7/site-packages/sphinx/writers/html.py", line 48, in translate
    self.document.walkabout(visitor)
  File "/usr/lib/python2.7/site-packages/docutils/nodes.py", line 174, in walkabout
    if child.walkabout(visitor):
  File "/usr/lib/python2.7/site-packages/docutils/nodes.py", line 174, in walkabout
    if child.walkabout(visitor):
  File "/usr/lib/python2.7/site-packages/docutils/nodes.py", line 174, in walkabout
    if child.walkabout(visitor):
  File "/usr/lib/python2.7/site-packages/docutils/nodes.py", line 166, in walkabout
    visitor.dispatch_visit(self)
  File "/usr/lib/python2.7/site-packages/docutils/nodes.py", line 1882, in dispatch_visit
    return method(node)
  File "/usr/lib/python2.7/site-packages/sphinx/writers/html.py", line 223, in visit_reference
    assert len(node) == 1 and isinstance(node[0], nodes.image)
AssertionError
```

And node being
```xml
 <reference classes="btn btn-primary btn-lg btn-block center-block" refuri="https://demo.odoo.com?module=base.action_res_partner_bank_account_form"><paragraph>View <emphasis>Bank Account</emphasis> in our Online Demonstration</paragraph></reference>
```

**Comment by xmo-odoo** [comment]
It's actually because in Sphinx 1.5 our tech of programmatically setting html_translator_class doesn't work, so the custom translator is ignored and the standard one is used indeed, which breaks.

Should be fixed by  95b64ca, which probably also needs to be applied in odoo/odoo.

**Comment by UnixedIT** [comment]
the problem return back on 

https://readthedocs.org/projects/documentation-user-arabia/builds/10217180/
